Core/HW/StereoResampler.cpp

```diff
@@ -266,15 +266,15 @@ void StereoResampler::PushSamples(const s32 *samples, unsigned int numSamples) {
 	u32 indexW = m_indexW.load();
 
 	u32 cap = m_maxBufsize * 2;
-	// If unthrottling, no need to fill up the entire buffer, just screws up timing after releasing unthrottle.
-	if (PSP_CoreParameter().unthrottle) {
+	// If unthrottling, no need to fill up the entire buffer, just screws up timing after releasing the fast-forward button.
```

Comment:
nit: If we're trying to change everywhere, there's still "unthrottling" here.

-[Unknown]